### PR TITLE
Jspm/TypeScript sample

### PR DESCRIPTION
**Pull request by vladima**

### CODE REVIEW

File: jspm/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "jspm": {
+    "directories": {},
+    "dependencies": {
+      "core-js": "npm:core-js@^0.9.4"
+    },
+    "devDependencies": {
+      "typescript": "github:mhegazy/typescript@v1.5-beta2"
```
Comment (mhegazy) [inline jspm/package.json]: can we use the official one now?

**Comment by vladima** [comment]
This is the version that is currently supported by jspm. I can update it after the next release of jspm

**Comment by vladima** [comment]
@mhegazy jspm uses [hardcoded locations](https://github.com/jspm/jspm-cli/blob/0.16.0-beta.2/lib/core.js#L48) for core components so now it will always use old version, updating typescript reference does not change the behavior. This is good meaning that I can put correct reference immediately.

**Comment by mhegazy** [comment]
Thanks! :+1:

**Pull request merged**